server/src/main/java/org/opensearch/index/IndexModule.java

```diff
@@ -148,6 +148,14 @@ public final class IndexModule {
         Property.NodeScope
     );
 
+    public static final Setting<List<String>> INDEX_STORE_HYBRID_MMAP_EXTENSIONS = Setting.listSetting(
```

Comment:
This is an expert parameter which, I suspect, few are going to touch. Can we javadoc this as `expert` with a description similar to above?

Reply:
@nknize sure can